notNeededPackages.json

```diff
@@ -6,6 +6,12 @@
             "sourceRepoURL": "https://github.com/betterwaysystems/packer",
             "asOfVersion": "1.1.3"
         },
+        {
+            "libraryName": "Active Directory Authentication Library for JavaScript",
```

Comment:
This is used when the definition file is part of the actual package, in this case you replaces this definition with a new one. So you don't need this.

Reply:
Ah, understood. I've removed my changes to the `notNeededPackages` file.